packages/figma-utils/src/commands/import-variables.spec.ts

```diff
@@ -62,14 +62,8 @@ describe("import-variables.ts", () => {
     expect(functions.generateAsCSS).toHaveBeenCalledTimes(2);
     expect(fs.writeFileSync).toHaveBeenCalledTimes(2);
 
-    expect(fs.writeFileSync).toHaveBeenCalledWith(
-      "test-cwd/test-file-name-test-mode-1.css",
-      "mock-css-file-content",
-    );
+    expect(fs.writeFileSync).toHaveBeenCalledWith("test-cwd/test.css", "mock-css-file-content");
```

Comment:
It is imported to keep the mode name in the file, otherwise e.g. density variables are lost and only "density.css" is imported instead of "density-compact.css", "density.default.css" and "density-cozy.css"